.travis.yml

```diff
@@ -0,0 +1,15 @@
+---
+language: python
+python:
+  - "2.7"
+  - "3.6"
+install:
+  - gem install mdl
```

Comment:
I know I was probably the one who started this bad habit of mixing Ruby and Python runtimes, so forgive me :bowing_man: but I think it would be better to use [pymarkdownlint](https://pypi.org/project/pymarkdownlint/).

Reply:
For me it looks like a dead project: https://github.com/jorisroovers/pymarkdownlint, as already discussed previously in fact ;)